.golangci.yml

```diff
@@ -9,6 +9,16 @@ run:
 linters-settings:
   goimports:
     local-prefixes: antrea.io/antrea
+  revive:
+    ignore-generated-header: false
+    severity: warning
+    confidence: 0.8
```

Comment:
I have no idea what that number mean or what a good value would be, but I guess we can see over time if we get too many false positives